<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsDialog</class>
 <widget class="QDialog" name="SettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>509</width>
    <height>398</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#SettingsDialog {
	background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,                                stop: 0 #ffcc00, stop: 1 #ffe680);

	color : white;
}
</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_6">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="styleSheet">
      <string notr="true">#tabWidget {
	background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,                                stop: 0 #ffcc00, stop: 1 #ffe680);

	color : white;
}
</string>
     </property>
     <property name="currentIndex">
      <number>2</number>
     </property>
     <widget class="QWidget" name="equalTab">
      <property name="styleSheet">
       <string notr="true">#equalTab {
	background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,                                stop: 0 #ffcc00, stop: 1 #ffe680);

	color : white;
}
</string>
      </property>
      <attribute name="title">
       <string>Equalizer</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QCheckBox" name="enable">
         <property name="text">
          <string>enable</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QSlider" name="g1">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="g2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="g3">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="g4">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="g5">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="g6">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="g7">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="visualTab">
      <property name="styleSheet">
       <string notr="true">#visualTab {
	background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,                                stop: 0 #ffcc00, stop: 1 #ffe680);

	color : white;
}
</string>
      </property>
      <attribute name="title">
       <string>Visualisasi</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QLabel" name="lblScale_3">
           <property name="text">
            <string>Brick Z-spacing</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="sldStepZ">
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>36</number>
           </property>
           <property name="pageStep">
            <number>1</number>
           </property>
           <property name="value">
            <number>1</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLabel" name="lblScale_2">
           <property name="text">
            <string>Floor Rotation X</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="sldFloorX">
           <property name="minimum">
            <number>-360</number>
           </property>
           <property name="maximum">
            <number>360</number>
           </property>
           <property name="pageStep">
            <number>10</number>
           </property>
           <property name="value">
            <number>0</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLabel" name="lblScale">
           <property name="text">
            <string>Scale</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="sldScale">
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>20</number>
           </property>
           <property name="pageStep">
            <number>1</number>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QCheckBox" name="chkMagnitude">
         <property name="text">
          <string>Use Magnitudes</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="chkMovingCamera">
         <property name="text">
          <string>Moving Camera</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="chkTranzient">
         <property name="text">
          <string>Tranzient</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="chkFlashlight">
         <property name="text">
          <string>Flashlight</string>
         </property>
        </widget>
       </item>
      </layout>
      <zorder></zorder>
      <zorder></zorder>
      <zorder></zorder>
      <zorder>chkFlashlight</zorder>
      <zorder>chkTranzient</zorder>
      <zorder>chkMovingCamera</zorder>
      <zorder>chkMagnitude</zorder>
     </widget>
     <widget class="QWidget" name="generalTab">
      <property name="styleSheet">
       <string notr="true">#generalTab {
	background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,                                stop: 0 #ffcc00, stop: 1 #ffe680);

	color : white;
}
</string>
      </property>
      <attribute name="title">
       <string>General</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_7">
       <item>
        <widget class="QCheckBox" name="checkBox">
         <property name="text">
          <string>KeyBoar Active</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
